File: PDFaCodigoJavaCSharpRustPython/PA_StopWatch/PythonCode/Vstopwatch.py
import tkinter as tk

class Vstopwatch(tk.Tk):
    def __init__(self):
        super().__init__()
        self.title("Stopwatch MVC")
        self.geometry("300x200")
        
        # Emular FlowLayout
        self.timeField = tk.Entry(self, font=("Arial", 24, "bold"), justify="center")
        self.timeField.insert(0, "00:00:00")
        self.timeField.config(state="readonly")
        self.timeField.pack(pady=20)
        
        btn_frame = tk.Frame(self)
        btn_frame.pack()
        
        self.startButton = tk.Button(btn_frame, text="Start")
        self.startButton.pack(side=tk.LEFT, padx=5)
        
        self.stopButton = tk.Button(btn_frame, text="Stop")
        self.stopButton.pack(side=tk.LEFT, padx=5)
        
        self.exitButton = tk.Button(btn_frame, text="Exit")
        self.exitButton.pack(side=tk.LEFT, padx=5)

    def getStartButton(self):
        return self.startButton

    def getStopButton(self):
        return self.stopButton

    def getExitButton(self):
        return self.exitButton

    def setTime(self, time_str):
        self.timeField.config(state="normal")
        self.timeField.delete(0, tk.END)
        self.timeField.insert(0, time_str)
        self.timeField.config(state="readonly")

File: PDFaCodigoJavaCSharpRustPython/PA_StopWatch/JavaCode/src/PA_StopWatch/Vstopwatch.java
package PA_StopWatch;

import javax.swing.*;
import java.awt.*;

public class Vstopwatch extends JFrame {
    private JButton startButton = new JButton("Start");
    private JButton stopButton = new JButton("Stop");
    private JButton exitButton = new JButton("Exit");
    private JTextField timeField = new JTextField("00:00:00");

    public Vstopwatch() {
        setTitle("Stopwatch MVC");
        setSize(300, 200);
        setDefaultCloseOperation(JFrame.EXIT_ON_CLOSE);
        setLayout(new FlowLayout());
        
        timeField.setEditable(false);
        timeField.setFont(new Font("Arial", Font.BOLD, 24));
        timeField.setHorizontalAlignment(JTextField.CENTER);
        
        add(timeField);
        add(startButton);
        add(stopButton);
        add(exitButton);
        
        setVisible(true);
    }

    public JButton getStartButton() {
        return startButton;
    }

    public JButton getStopButton() {
        return stopButton;
    }

    public JButton getExitButton() {
        return exitButton;
    }

    public void setTime(String time) {
        timeField.setText(time);
    }
}